packages/core/core-flows/src/definition/cart/workflows/complete-cart.ts

```diff
@@ -80,11 +81,32 @@ export const completeCartWorkflow = createWorkflow(
     )
 
     const cartToOrder = transform({ input, cart }, ({ cart }) => {
-      const itemAdjustments = (cart.items || [])
-        ?.map((item) => item.adjustments || [])
+      const allItems = (cart.items ?? []).map((item) => {
+        return prepareLineItemData({
+          item,
+          variant: item.variant,
+          unitPrice: item.raw_unit_price ?? item.unit_price,
+          isTaxInclusive: item.is_tax_inclusive,
+          quantity: item.raw_quantity ?? item.quantity,
+          metadata: item?.metadata ?? {},
+          taxLines: item.tax_lines || [],
+          adjustments: item.adjustments || [],
+        })
+      })
+
+      const shippinMethods = JSON.parse(
+        JSON.stringify(cart.shipping_methods ?? []),
+        (_, value) => {
+          delete value.id
+          return value
+        }
+      )
```

Comment:
nit: typo
```suggestion
      const shippingMethods = JSON.parse(
        JSON.stringify(cart.shipping_methods ?? []),
        (_, value) => {
          delete value.id
          return value
        }
      )
```

suggestion: The stringify -> parse is a bit hard to read. Would maybe prefer a loop here. Also, didn't this work previously? What happened

Reply:
Just changed to make it explicity.
It was working, but not removing the `id`s the entities were reusing the same id from the cart.